prac_06/guitars.py

```diff
@@ -0,0 +1,48 @@
+"""
+Store a list of guitars and print details.
+"""
+from prac_06.guitar import Guitar
```

Comment:
I've noticed you don't have your guitar.py for your guitar class in your repo. Make you you put that in before Wednesday :)

Reply:
Well spotted - thanks Matt :)